Formula/leafnode.rb

```diff
@@ -14,6 +14,89 @@ def install
                           "--with-user=#{ENV["USER"]}", "--with-group=admin",
                           "--sysconfdir=#{etc}/leafnode", "--with-spooldir=#{var}/spool/news/leafnode"
     system "make", "install"
+    (prefix/"homebrew.mxcl.fetchnews.plist").write fetchnews_plist
+    (prefix/"homebrew.mxcl.texpire.plist").write texpire_plist
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
I'm happy to ship this as-is if @DomT4 is but @rakkesh a PR to handle multiple plists would be great.